src/_guides/language/coming-from/js-to-dart.md

```diff
@@ -1018,7 +1017,7 @@ work in the same way:
 
 Unlike JavaScript,
 Dart doesn’t have the concept of abstract equality,
-so the `==` and `!=` operators aren't available.
+so the `==` and `!=` JavaScript operators have no equivalent.
```

Comment:
I made this adjustment as it was confusing to say the "`==` and `!=`" aren't available, as they are. It's just the ones from JS have no equivalent in Dart. Open to suggestions on how to best word this :)

Reply:
lgtm